src/pages/TeamOverview/TeamOverview.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {useLocation, useParams} from 'react-router-dom';
+import {ListItem, UserData} from 'types/types';
+import {getUserColumns} from 'utils/utils';
+import {getTeamOverview, getUserData} from '../../api/api';
+import Card from '../../components/Card/Card';
+import {GlobalContainer} from '../../components/global.styled';
+import Header from '../../components/Header/Header';
+import List from '../../components/List/List';
+
+const getUsersCards = (users: UserData[]): ListItem[] => {
```

Comment:
renamed from `mapArray` to `getUsersCards` , to allow for better understatement of the what the method does